site/content/en/docs/Tutorials/nvidia_gpu.md

```diff
@@ -93,7 +93,7 @@ to expose GPUs with `--vm-driver=kvm2`. Please don't mix these instructions.
 
 - Install NVIDIA's device plugin:
   ```shell
-  kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.10/nvidia-device-plugin.yml
+  kubectl create -f https://github.com/NVIDIA/k8s-device-plugin/blob/master/nvidia-device-plugin.yml
```

Comment:
this needs to be reffering to raw file other wise it will return html
please change it to 

https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/master/nvidia-device-plugin.yml

Reply:
i have changed to :https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/master/nvidia-device-plugin.yml